specification/common-types/resource-management/v6/managedidentitywithdelegation.json

```diff
@@ -0,0 +1,54 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Common types",
+    "version": "6.0"
+  },
+  "paths": {},
+  "definitions": {
+    "DelegatedResource": {
+      "type": "object",
+      "description": "Delegated resource properties - internal use only.",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The ARM resource id of the delegated resource - internal use only."
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

Why isn't it format: "arm-id"?

Reply:
Since manageIdentityWithDelegation is not currently available in v6, we did not make any changes to v5. It is already defined this way in v5:  https://github.com/Azure/azure-rest-api-specs/blob/2d3a19a9754e340760eed074ebd75987585ce0bd/specification/common-types/resource-management/v5/managedidentitywithdelegation.json#L13

Reply:
So we are preferring not to fix it because its considered breaking?